folly/FBVector.h

```diff
@@ -1748,4 +1748,11 @@ void attach(fbvector<T, A>& v, T* data, size_t sz, size_t cap) {
   v.impl_.z_ = data + cap;
 }
 
+#if __cpp_deduction_guides >= 201703 || _MSC_VER
```

Comment:
You may hate me now, but based on the discussion in the internal diff review tool, there appears to be precisely one point release of MSVC which supports deduction guides but which also leaves the name `__cpp_deduction_guides` undefined. So we can remove `_MSC_VER` from the guard.

Would you mind also submitting a PR to `Replaceable` removing `_MSC_VER` from the guard?

Reply:
No worries, I don't hate you 😆 Thanks for the heads up. I lean on you guys for the MSVC nuances as I just use Clang for local dev.

Just removed `_MSC_VER` from the macro guard. [Here](https://github.com/facebook/folly/pull/1097) is a PR for removing `_MSC_VER` from the guard as well for `Replaceable`'s deduction guide.